runtime/onert/core/src/ir/OperationValidator.cc

```diff
@@ -285,8 +285,10 @@ void OperationValidator::visit(const operation::ElementwiseUnary &node)
   }
   else if (node.param().op_type == operation::ElementwiseUnary::Type::QUANTIZE)
   {
-    OP_REQUIRES(isValidType(input_index, DataType::FLOAT32));
-    OP_REQUIRES(isValidType(output_index, DataType::QUANT_UINT8_ASYMM));
+    OP_REQUIRES(isValidType(
+      input_index, {DataType::FLOAT32, DataType::QUANT_UINT8_ASYMM, DataType::QUANT_INT8_ASYMM}));
```

Comment:
TFLite 2.3 allow `int16` input (`int16` -> `int16` or `int8`)

Reply:
@hseok-oh Thank you for information. But `onert` does not support them yet. I allow the cases that `onert` can currently handle, not tflite's latest specification.

Reply:
I see. It's OK.